types/es-abstract/scripts/collect-intrinsics.ts

```diff
@@ -3,304 +3,36 @@
 import path = require('path');
 import fs = require('fs');
 
-import { BaseIntrinsic, Override, BASE_INTRINSICS, LEGACY_ALIASES, BASE_INTRINSIC_DATA } from './intrinsics-data';
+import { BASE_INTRINSICS, LEGACY_ALIASES, BASE_INTRINSIC_DATA } from './intrinsics-data';
```

Comment:
can the types for GetIntrinsic in es-abstract not refer to the get-intrinsic types, rather than having to generate them in two places?

Reply:
Currently, this defines the types for `es‑abstract@1.17.x`, but `es‑abstract` only uses `get‑intrinsic` since `es‑abstract@1.18.x‑*`, which is a pre‑release version, and AFAIK, **DefinitelyTyped** doesn’t support creating types for pre‑release versions.

Reply:
Also, `@types/es‑abstract` supports **TypeScript 3.8**, but `@types/get‑intrinsic` requires **TypeScript 3.9+**.

Reply:
The first point makes sense to me; altho it shouldn’t be hard to make the two packages require the same typescript version.

Reply:
@ljharb
> it shouldn’t be hard to make the two packages require the same typescript version.

I’d rather not do that in a patch release.